backend/src/main/java/com/redhat/cloud/notifications/routers/NotificationResource.java

```diff
@@ -232,7 +232,7 @@ public List<Facet> getApplicationsFacets(@Context SecurityContext sec, @QueryPar
     @GET
     @Path("/facets/bundles")
     @Produces(APPLICATION_JSON)
-    @Operation(summary = "Return a thin list of configured bundles. This can be used to configure a filter in the UI")
+    @Operation(summary = "List configured bundles", description = "Returns a thin list of configured bundles that includes the bundle name, the display name, and the ID. You can use this list to configure a filter in the UI.")
```

Comment:
Will users understand the term, "thin list?"

Reply:
good catch, we should be OK by removing `thin` as we later explain what fields are included.